src/plugins/kibana_react/public/validated_range/validated_dual_range.tsx

```diff
@@ -47,7 +66,11 @@ export class ValidatedDualRange extends Component {
     return null;
   }
 
-  _onChange = value => {
+  state: State = {
+    value: ['0', '0'],
+  };
+
+  _onChange = (value: any) => {
```

Comment:
value is `ValueMember[]` instead of `any`

Reply:
I just decided to pull in `Value` from `EuiDualRangeProps` the same way we pulled in `ValueMember`